tests/engines/collections/acceptance/submit/submit-test.ts

```diff
@@ -30,33 +33,164 @@ module('Collections | Acceptance | submit', hooks => {
             primaryCollection,
             licensesAcceptable,
         });
+
+        const store = this.owner.lookup('service:store');
+
+        const newTitle = 'New Title';
+        const newDescription = 'New description.';
+
         await visit(`/collections/${provider.id}/submit`);
+
+        /* Select a project */
+
         await percySnapshot('Collections | Acceptance | submit | select project');
+
+        // open item picker
         await untrackedClick('[data-test-collections-item-picker] .ember-power-select-trigger');
-        await untrackedClick(`[data-test-collections-node-title="${nodeToBeAdded.title}"]`);
+        // select node
+        const nodeToBeAddedOption = document.querySelector(
+            `[data-test-collections-node-title="${nodeToBeAdded.title}"]`,
+        );
+        if (nodeToBeAddedOption) {
+            await untrackedClick(nodeToBeAddedOption);
+        }
+
+        /* Project metadata */
 
         await percySnapshot('Collections | Acceptance | submit | project metadata');
-        await untrackedClick('[data-test-collections-license-picker] .ember-power-select-trigger');
-        await untrackedClick('.ember-power-select-option');
-        await untrackedClick('[data-test-project-metadata-continue]');
+
+        await fillIn('[data-test-project-metadata-title] input', newTitle);
+        await fillIn('[data-test-project-metadata-description] textarea', newDescription);
+        // open license picker
+        await untrackedClick('[data-test-project-metadata-license-picker] .ember-power-select-trigger');
+        // select first license
+        const firstLicenseOption = document.querySelector('.ember-power-select-option');
+        if (firstLicenseOption) {
```

Comment:
Similar issue with this condition. Is there a case in which this would be false?

Reply:
SAA